libraries/standard/http/src/private/http_client_parse.h

```diff
@@ -0,0 +1,72 @@
+#ifndef HTTP_CLIENT_PARSE_H_
+#define HTTP_CLIENT_PARSE_H_
+
+#include "http_client.h"
+/* #include "http_parser.h" */
```

Comment:
Commented out code

Reply:
This is going to be implemented in the next PR.